experimental/Schemes/SpaceGerms.jl

```diff
@@ -645,7 +645,7 @@ Spectrum
 julia> phi
 Affine scheme morphism
   from [x1, x2, x3]  Spec of localization of Q at complement of maximal ideal
-  to   [x1, x2, x3]  V(x1^2 + x2^2 + x3^2)
+  to   [x1, x2, x3]  scheme(x1^2 + x2^2 + x3^2)
```

Comment:
Here we are disclosing schemes, which are the underlying technology,  to a non-scheme community. For them this represents a morphism of germs of complex spaces. This can certainly be confusing for them.

But as I see no alternative, we leave it as it is now...

Reply:
You could implement a custom printing method for space germs?

Reply:
Indeed, this is the way in which I should solve it.